### PR TITLE
Approve deny fix

### PR DESCRIPTION
**Pull request by alexhung**
- Fix RT status not being set when validation fails
- Reformat python code using Black
- Add volume binding for code files when running in docker
- Add logging to scripts
- Update README and DEBUG
- Update .gitignore

### CODE REVIEW

**Review by vinayagg — APPROVED** [review]

File: governance/approveDeny/DEBUG.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ Remove Artifactory docker container:
 Sometimes you have to go digging around the Artifactory docker container:
 
   ```
-  docker exec -it --user root rt1 sh
+  docker exec -it --user root rt1 bash
```
Comment (chb0github) [inline governance/approveDeny/DEBUG.md:163]: why?

File: governance/approveDeny/README.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ Installation
...
   * [HTTPBuilder](https://bintray.com/bintray/jcenter/org.codehaus.groovy.modules.http-builder%3Ahttp-builder/_latestVersion)
   * [Json-lib](https://bintray.com/bintray/jcenter/net.sf.json-lib%3Ajson-lib/_latestVersion)
   * [Xml-resolver](https://bintray.com/bintray/jcenter/xml-resolver%3Axml-resolver/_latestVersion)
 1. Edit the `approveDeny.json` file. See the example `approveDeny.json` provided or
    the details below on how to edit `approveDeny.json`.
-1. Copy `approveDeny.groovy` and `approveDeny.json` to the `${ARTIFACTORY_HOME}/etc/artifactory/plugins` directory.
+1. Copy `approveDeny.groovy` and `approveDeny.json` to the `${ARTIFACTORY_HOME}/var/etc/artifactory/artifactory/plugins` directory.
 1. There are two ways to load the plugin:
-   - Manually force the plugin to be loaded `curl -u username:password -XPOST -H'Content-Type: application/json' http://localhost:8080/artifactory/api/plugins/reload`
+   - Manually force the plugin to be loaded `curl -u username:password -XPOST -H'Content-Type: application/json' http://localhost:8082/artifactory/api/plugins/reload`
```
Comment (chb0github) [inline governance/approveDeny/README.md:42]: I aways recommend referring to [netrc](https://everything.curl.dev/usingcurl/netrc) as it doesn't encourage people to put username and pass on the CLI:

`curl -sn ...`

File: governance/approveDeny/DEBUG.md (hunk 1 above)
Comment (alexhung) [reply governance/approveDeny/DEBUG.md:163]: Convenient, mainly. Autocomplete is nice when navigating directories 😄

File: governance/approveDeny/scripts/rundocker.sh [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -it --name rt1 -p 8081:8081 -p 8082:8082 -p 5555:5555 -e "JAVA_TOOL_OPTIONS=\"-agentlib:jdwp=transport=dt_socket,address=*:5555,server=y,suspend=n\"" docker.bintray.io/jfrog/artifactory-pro:latest /bin/bash
+docker run -it --rm --name rt1 -p 8081:8081 -p 8082:8082 -p 5555:5555 -e "JAVA_TOOL_OPTIONS=\"-agentlib:jdwp=transport=dt_socket,address=*:5555,server=y,suspend=n\"" -v $(pwd)/approveDeny.groovy:/opt/jfrog/artifactory/var/etc/artifactory/plugins/approveDeny.groovy -v $(pwd)/approveDeny.json:/opt/jfrog/artifactory/var/etc/artifactory/plugins/approveDeny.json docker.bintray.io/jfrog/artifactory-pro:latest /bin/bash
```
Comment (chb0github) [inline governance/approveDeny/scripts/rundocker.sh:3]: I thought bintray was going away

Comment (chb0github) [reply governance/approveDeny/scripts/rundocker.sh:3]: Also, I think you should use docker-compose for this - it's meant to capture extensive docker config

Comment (alexhung) [reply governance/approveDeny/scripts/rundocker.sh:3]: Good catch, I'll try to find the equivalent and update the URL.

File: governance/approveDeny/README.md (hunk 2 above)
Comment (alexhung) [reply governance/approveDeny/README.md:42]: I'll update the doc to include netrc

**Review by chb0github — APPROVED** [review]
Nothing major, just some suggestions

**Comment by alexhung** [comment]
@chb0github Updated pushed

**Pull request merged**